src/Abp.EntityFrameworkCore/EntityFrameworkCore/DefaultDbContextResolver.cs

```diff
@@ -101,5 +100,10 @@ private object CreateOptionsForType(Type dbContextType, string connectionString,
 
             throw new AbpException($"Could not resolve DbContextOptions for {typeof(TDbContext).AssemblyQualifiedName}.");
         }
+
+        protected virtual void CreateConfiguration<TDbContext>(AbpDbContextConfiguration<TDbContext> configuration) where TDbContext : DbContext
```

Comment:
Please rename this method to ReplaceServices

Reply:
Done